gr-digital/lib/map_bb_impl.cc

```diff
@@ -49,18 +49,19 @@ void map_bb_impl::set_map(const std::vector<int>& map)
 {
     gr::thread::scoped_lock guard(d_mutex);
 
-    for (int i = 0; i < 0x100; i++)
+    for (int i = 0; i < d_map_size; i++)
```

Comment:
I have no idea why this is set this way. I guess it's just to make sure there's -some- value set for every entry? I'd say to leave this alone, since it's worked this way all along ...